package/run.ps1

```diff
@@ -46,6 +46,22 @@ function Get-StringHash {
     return (Get-FileHash -InputStream $stringAsStream -Algorithm SHA256).Hash.ToLower()
 }
 
+# stops the RKE2 process as well as other processes spawned from rke2
+function Stop-RKE2-Processes () {
+    $ProcessNames = @('rke2', 'kube-proxy', 'kubelet', 'containerd', 'calico-node')
+    foreach ($ProcessName in $ProcessNames) {
+        Write-LogInfo "Checking if $ProcessName process exists"
+        if ((Get-Process -Name $ProcessName -ErrorAction SilentlyContinue)) {
```

Comment:
nit: any reason for two parentheses here?

Reply:
It seemed to be a style done by Ross in a few powershell scripts, but after testing it there doesn't seem to be a functional difference between one set and two sets of parentheses. Dropped the second set